src/dialogxml/widgets/control.cpp

```diff
@@ -210,7 +210,7 @@ void cControl::setActive(bool active) {
 	depressed = active;
 }
 
-void cControl::setFormat(eFormat prop, short val) {
+void cControl::setFormat(eFormat prop, sf::Uint8 val) {
```

Comment:
I am a _little_ bit suspicious about this change to `sf::Uint8`. Are you certain that all existing uses fit into a single byte?

Reply:
Hmm, seems like eControlType is outside the scope. Casting this short to sf::Uint8 will essentially do the same and cut overflowing values. I've made the change, because my gcc refused to compile it. Will check other options.

Reply:
I added a cast to sf::Uint8, should be always in one byte range:

`newVal = sf::Color{(sf::Uint8)val, (sf::Uint8)val, (sf::Uint8)val};`

It compiles.